cime_config/testdefs/testlist_clm.xml

```diff
@@ -1524,13 +1524,13 @@
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
-  <test name="ERP_P64x2_Lm25" grid="f10_f10_mg37" compset="I2000Clm60BgcCrop" testmods="clm/monthly">
+  <test name="ERP_P64x2_Ly36" grid="f10_f10_mg37" compset="I2000Clm60BgcCrop" testmods="clm/monthly">
```

Comment:
Need to move this back to previous length, as it increases wallclock time for the test and it runs into the limit even with increasing it by an hour.

So will use _Ld762

Reply:
Why did you change it to LY36 instead of Ly3???

Reply:
Oh, no wonder is was taking forever! Good catch.

The 36 was unintentional, I must have been thinking in terms of months in 3 years...

Reply:
@ekluzek - I haven't looked at these specific tests in a while, but keep in mind that some tests were deliberately set up to restart mid-year. So it's important to consider whether you want _Lm36 or _Ly3, since they give different behavior in terms of when they restart. (Maybe you already have considered this.)

Reply:
@billsacks thanks for that point. I kept some of the ER tests to the same length, so they'll be fine. But, you have a good point about not restarting on the year boundary, since that's already the standard default. I might adjust a couple in regard to that.

Reply:
OK, there's five tests this applies to:

``` diff
-  <test name="ERS_Lm54_Mmpi-serial" grid="1x1_numaIA" compset="I2000Clm50BgcCrop" testmods="clm/ciso_monthly_matrixcn_spinup">
+  <test name="ERS_Ly4_Mmpi-serial" grid="1x1_numaIA" compset="I2000Clm50BgcCrop" testmods="clm/ciso_monthly_matrixcn_spinup">
-  <test name="ERP_P64x2_Lm36" grid="f10_f10_mg37" compset="I2000Clm50BgcCrop" testmods="clm/clm50cropIrrigMonth_interp">
+  <test name="ERP_P64x2_Ly3" grid="f10_f10_mg37" compset="I2000Clm50BgcCrop" testmods="clm/clm50cropIrrigMonth_interp">
-  <test name="ERS_P128x1_Lm25" grid="f10_f10_mg37" compset="I2000Clm60Fates" testmods="clm/FatesColdNoComp">
+  <test name="ERS_P128x1_Ly3" grid="f10_f10_mg37" compset="I2000Clm60Fates" testmods="clm/FatesColdNoComp">
-  <test name="ERS_Lm25" grid="1x1_smallvilleIA" compset="IHistClm50BgcCropQianRs" testmods="clm/smallville_dynlakes_monthly">
+  <test name="ERS_Ly3" grid="1x1_smallvilleIA" compset="IHistClm50BgcCropQianRs" testmods="clm/smallville_dynlakes_monthly">
-  <test name="ERS_Lm25" grid="1x1_smallvilleIA" compset="IHistClm50BgcCropQianRs" testmods="clm/smallville_dynurban_monthly">
+  <test name="ERS_Ly3" grid="1x1_smallvilleIA" compset="IHistClm50BgcCropQianRs" testmods="clm/smallville_dynurban_monthly">
```

So I'll change _Lm54 to _Ld1640, Lm25 to Ld761 (762 for 2000), and Lm36 to Ld1096